mlirlib/src/IR.scala

```diff
@@ -147,6 +146,13 @@ given LocationApi with
     inline def locationGetContext(
       using arena: Arena
     ): Context = Context(mlirLocationGetContext(arena, location.segment))
+    inline def print(
```

Comment:
```suggestion
    inline def locationPrint(
```
for matching the C-API name.

Reply:
we should omit the type prefix in corresponding extension methods